modules/apps/users-admin/users-admin-test/src/testIntegration/java/com/liferay/users/admin/indexer/test/UserIndexerIndexedFieldsTest.java

```diff
@@ -0,0 +1,143 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.users.admin.indexer.test;
+
+import com.liferay.arquillian.extension.junit.bridge.junit.Arquillian;
+import com.liferay.portal.kernel.model.User;
+import com.liferay.portal.kernel.search.Document;
+import com.liferay.portal.kernel.search.Field;
+import com.liferay.portal.kernel.test.rule.AggregateTestRule;
+import com.liferay.portal.kernel.test.rule.SynchronousDestinationTestRule;
+import com.liferay.portal.kernel.util.StringUtil;
+import com.liferay.portal.search.test.util.FieldValuesAssert;
+import com.liferay.portal.test.rule.LiferayIntegrationTestRule;
+import com.liferay.portal.test.rule.PermissionCheckerTestRule;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * @author Luan Maoski
+ */
+@RunWith(Arquillian.class)
+public class UserIndexerIndexedFieldsTest extends BaseUserIndexerTestCase {
+
+	@ClassRule
+	@Rule
+	public static final AggregateTestRule aggregateTestRule =
+		new AggregateTestRule(
+			new LiferayIntegrationTestRule(),
+			PermissionCheckerTestRule.INSTANCE,
+			SynchronousDestinationTestRule.INSTANCE);
+
+	@Before
+	public void setUp() throws Exception {
+		super.setUp();
+	}
+
+	@Test
+	public void testIndexedFields() throws Exception {
+		String firstName = "richard";
+		String lastName = "bower";
+		String screenName = "richard bower";
+
+		User user = userFixture.createAnUser(firstName, lastName, screenName);
+
+		String searchTerm = "richard";
+
+		Document document = userIndexerFixture.searchOnlyOne(searchTerm);
+
+		indexedFieldsFixture.postProcessDocument(document);
+
+		Map<String, String> expected = _expectedFieldValues(user);
+
+		FieldValuesAssert.assertFieldValues(expected, document, searchTerm);
+	}
+
+	private Map<String, String> _expectedFieldValues(User user)
+		throws Exception {
+
+		Map<String, String> map = new HashMap<>();
+
+		map.put(Field.ENTRY_CLASS_PK, String.valueOf(user.getUserId()));
+		map.put(Field.ENTRY_CLASS_NAME, User.class.getName());
+		map.put(Field.COMPANY_ID, String.valueOf(user.getCompanyId()));
+
+		map.put("firstName", StringUtil.toLowerCase(user.getFirstName()));
```

Comment:
@luanmaoski, If I'm not wrong, only "*_sortable" fields are lower case.

Reply:
You are right. I will change that. The only field that needs `lowerCase` and isn't sortable is `username`